src/icalendar/prop/__init__.py

```diff
@@ -3030,13 +3031,13 @@ def to_jcal(self, name: str) -> list:
 
     @classmethod
     def from_jcal(cls, jcal_property: list) -> Self:
-        """Parse jcal from :rfc:`7265`.
+        """Parse jCal from :rfc:`7265`.
 
         Args:
-            jcal_property: The jcal property to parse.
+            jcal_property: The jCal property to parse.
 
         Raises:
-            JCalParsingError: If the jcal provided is invalid.
```

Comment:
@stevepiercy, We should probably add a vale spell check for jcal - I mean jCal ;)

I vale running on the source or the HTML output? Because on the HTML output, it would check the source code, too.

Reply:
You mean this? 😀

https://github.com/collective/icalendar/pull/979/files#diff-7e9085154b26dfc128ab0cb0c6f07c8b905119ceb727c6f7e63424f0c1b8420eR34

Vale is configured to check .rst source, not HTML output. Vale is not very good at HTML linting, yielding both false positives and false negatives.

- https://vale.sh/docs/formats/rst
- https://vale.sh/docs/formats/html

However, it may be useful to associate docstrings with reStructuredText.

- https://vale.sh/docs/formats/code

If you think that's useful, then turn this comment into an issue by clicking the `...` and selecting "Reference in new issue".

Reply:
Issue created! This can be marked as resolved for me.

Reply:
See https://github.com/collective/icalendar/issues/1007